deploy/constants.js

```diff
@@ -1,14 +1,12 @@
-// ZepCore
+// Kernel
 export const NEW_VERSION_COST = 2;
 export const DEVELOPER_FRACTION = 10;
 export const ZEPPELIN_ACCOUNT = web3.eth.accounts[2];
-export const DEVELOPER_ACCOUNT = web3.eth.accounts[2];
+export const KERNEL_VERSION = "1.0";
```

Comment:
It should be part of the release right? I guess we can move it down to that section